src/quicklist.c

```diff
@@ -476,12 +513,32 @@ REDIS_STATIC int _quicklistNodeAllowMerge(const quicklistNode *a,
         (node)->sz = ziplistBlobLen((node)->zl);                               \
     } while (0)
 
+static void __quicklistInsertEntryNode(quicklist *quicklist, quicklistNode *old_node,
+                                       void *value, size_t sz, bool after)  {
+
+
```

Comment:
do you mean replacing QUICKLIST_NODE_CONTAINER_NONE with QUICKLIST_NODE_CONTAINER_NONE_PLAIN 
or adding a new enum QUICKLIST_NODE_PLAIN ?

Reply:
i think we can't use `QUICKLIST_NODE_PLAIN` since this is an enum that describes the `->container` field. so the prefix of this enum is needed to be common.
i think that `NONE` and `PLAIN` are in some way synonymous, and if we use the term `PLAIN` and we can more easily refer to is and search for it (unlike "NONE").
so i think `QUICKLIST_NODE_CONTAINER_PLAIN` is the right term.
@sundb WDYT?